seq2science/schemas/config/scrna.schema.yaml

```diff
@@ -29,5 +29,81 @@ properties:
+    properties:
+      method: 
+        description: method -- experimental method
+        enum:  [384plate,droplet]
+      barcode_file:  
+        description: "barcode_file -- .tab delimited file with 2-col layout for plate based methods (with DNA barcode and well-id)"
+        type: string
+      mt_genes_file: 
+        description: "mt_genes_file -- .txt file containing one column with mitchondrial genes"
+        type: string
+      isvelo:
+         description: "is_velo -- velocity workflow"
+         type: boolean
+      run.sct:
+         description: "run.sct -- perform SCTransform normalization, FALSE runs log(p1) NormalizeData, HVG selection and ScaleData in Seurat"
+         type: boolean
+      run.jackstraw:
+         description: "run.jackstraw -- perform Jackstraw analysis (does not work with SCTransform)"
+         type: boolean    
+      add.spikes.ercc: 
+        label: "add.spikes.ercc -- use ERCC RNA spike-ins (if present)"
+        type: boolean
+      ercc_pct_max: 
+        description: "ercc_pct_max -- max percentage ERCC spike-ins counts"
+        type: integer
+      add.spikes.mt: 
+        description: "add.spikes.mt -- use mitochondrial gene-list for qc"
+        type: boolean
+      mt_pct_max: 
+        description: "mt_pct_max -- max percentage mitochondrial genes counts"
+        type: integer
+      gene_tresh: 
+        description: "gene_tresh -- gene filter: threshold for genes considered expressed"
+        type: integer
+      amount_cells_expr: 
+        description: "amount_cells_expr -- gene filter: threshold for minimal amount of cells a gene should be expressed in"
+        type: integer
+      total_counts_tresh: 
+        description: "total_counts_tresh -- cell filter: threshold for minimal amount of UMI counts detected in a cell"
+        type: integer
+      total_feat_tresh: 
+        description: "total_feat_tresh -- cell filter: threshold for minimal amount of features (genes) detected in a cell"
+        type: integer
+      nhvg: 
+        description: "nhvg -- number of Highly Variable Genes, used in Seurat's FindVariableFeatures"
+        type: integer
+      pcs_for_overview:
+        description: "pcs_for_overview -- principal components for overview in combined umap plot, runs UMAP for PC 1-value"
+        type: string
+      pcs_max_hvg: 
+        description: "pcs_max_hvg -- max number of principal components to visualize"
+        type: integer  
+      default:
+        method: 384plate
+        barcode_file: ""
+        mt_genes_file: ""
+        isvelo: true
+        run.sct: true
+        run.jackstraw: false
+        add.spikes.ercc : true
+        add.spikes.mt: true
+        ercc_pct_max: 20
+        mt_pct_max: 50
+        gene_tresh: 0
+        amount_cells_expr: 0  
+        total_counts_tresh: 1000
+        total_feat_tresh: 500
+        nhvg: 2000
+        pcs_max_hvg: 70
+        pcs_for_overview: 10,20,30,40,50
```

Comment:
minor stuff, but it's best for files to end with a newline: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline